packages/ogmios/src/CardanoNode/OgmiosCardanoNode.ts

```diff
@@ -1,96 +1,105 @@
 import {
-  Cardano,
   CardanoNode,
   EraSummary,
   GeneralCardanoNodeError,
   GeneralCardanoNodeErrorCode,
   HealthCheckResponse,
+  Milliseconds,
   StakeDistribution
 } from '@cardano-sdk/core';
-import {
-  ConnectionConfig,
-  LedgerStateQuery,
-  createConnectionObject,
-  createLedgerStateQueryClient,
-  getServerHealth
-} from '@cardano-ogmios/client';
 import { Logger } from 'ts-log';
-import { RunnableModule, contextLogger } from '@cardano-sdk/util';
-import { createInteractionContextWithLogger, ogmiosServerHealthToHealthCheckResponse } from '../util';
-import { queryEraSummaries, withCoreCardanoNodeError } from './queries';
+import {
+  MonoTypeOperatorFunction,
+  Observable,
+  Subject,
+  defaultIfEmpty,
+  firstValueFrom,
+  pipe,
+  takeUntil,
+  throwError,
+  throwIfEmpty,
+  timeout
+} from 'rxjs';
+import { OgmiosObservableCardanoNode } from './OgmiosObservableCardanoNode';
+import { RunnableModule } from '@cardano-sdk/util';
+
+const DEFAULT_RESPONSE_TIMEOUT = Milliseconds(10_000);
```

Comment:
Would be better if this was configurable by the app (cardano-services), but hardcoded will do the job for now